go/buildkit.md

```diff
@@ -0,0 +1,6 @@
+---
+title: How to enable BuildKit
+description: Instructions on enabling BuildKit
+keywords: BuildKit, docker build, configuration
```

Comment:
I _think_ these are redundant as they're not rendered anywhere for redirects. That said, they provide a bit of details as to what the redirect is for, so didn't really hurt to have them here.